src/pages/index.jsx

```diff
@@ -1,21 +1,85 @@
 import React from 'react'
-import Button from '../components/Button'
-import Card from '../components/Card'
-import Hero from '../components/Hero'
 import Layout from '../components/Layout'
+import Hero from '../components/Hero'
+import ContentSection from '../components/ContentSection'
+import ContactUs from '../components/ContactUs'
+import Organizations from '../images/Organizations.png'
+import Volunteers from '../images/Volunteers.png'
+import Heading from '../components/Heading'
+import IconGrid from '../components/IconGrid'
+import styled from 'styled-components'
+import anyone from '../images/icons/anyone.svg'
+import communityorganizers from '../images/icons/communityorganizers.svg'
+import datascientists from '../images/icons/datascientists.svg'
+import designers from '../images/icons/designers.svg'
+import developers from '../images/icons/developers.svg'
+import productmanagers from '../images/icons/productmanagers.svg'
+import projectmanagers from '../images/icons/projectmanagers.svg'
+import socialworkers from '../images/icons/socialworkers.svg'
+
+const WorkWithUs = styled.section`
+  margin-bottom: 4rem;
+`
 
 const IndexPage = () => {
+  const icons = [
+    { img: developers, title: 'Developers' },
+    { img: designers, title: 'Designers' },
+    { img: datascientists, title: 'Data Scientists' },
+    { img: projectmanagers, title: 'Project Managers' },
+    { img: productmanagers, title: 'Product Managers' },
+    { img: communityorganizers, title: 'Community Organizers' },
+    { img: socialworkers, title: 'Social Workers' },
+    { img: anyone, title: 'Anyone' },
+  ]
+
   return (
     <Layout>
-      <h2>homepage</h2>
       <Hero />
-      <Card />
-      <Button text={'Submit'} small arrow />
-      <Button text={'Submit'} arrow />
-      <Button text={'Button'} secondary small />
-      <Button text={'Button'} secondary />
-      <Button text={'Link'} textBtn arrow link small />
-      <Button text={'Join Us on Slack'} textBtn arrow link />
+      <WorkWithUs>
+        <Heading
+          headingText="Work With Us"
```

Comment:
This text looks bolder in the Figma file 
https://www.figma.com/file/6ipIKBMMJL2TgWBzztA6lg/Handshake---Brigade-Design-System?node-id=1018%3A1059&t=mHbNkDyhYR5WiYdx-1

Reply:
I made a separate ticket for this. Thanks for pointing it out.